src/raftstore/store/peer.rs

```diff
@@ -1168,6 +1168,12 @@ impl Peer {
 
         self.add_ready_metric(&ready, &mut ctx.raft_metrics.ready);
 
+        if !ready.committed_entries.as_ref().map_or(true, Vec::is_empty)
+            && ctx.current_time.is_none()
+        {
+            ctx.current_time.replace(monotonic_raw_now());
```

Comment:
Why not initialize it at L1285?

Reply:
It look odd to initialize `current_time ` here, but it based on a reason:
when there are proposal msg `current_time` will be initialize at L1650, and when there are not, initialize `current_time` here is more close to initialize at L1650 than L1285, as between here and L1650 there is no time consume work like send msg or write to disk.